core/domain/draft_upgrade_services.py

```diff
@@ -103,6 +103,19 @@ def try_upgrading_draft_to_exp_version(
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v43_dict_to_v44_dict(cls, draft_change_list):
+        """Converts draft change list from state version 42 to 43.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        return draft_change_list
```

Comment:
Maybe add a line comment explaining why just returning the unchanged change list is OK.

Reply:
Done.